

----------------------------------------------------------------------------
Training STG with config/adult.yml in env torch

WARNING:root:imports error 
 You need to install pymongo>=3.9.0 in order to use MongoOutput 
Namespace(config='config/adult.yml', model_name='STG', dataset='Adult', objective='binary', use_gpu=False, gpu_ids=[0, 1], data_parallel=True, optimize_hyperparameters=False, n_trials=2, direction='maximize', num_splits=5, shuffle=True, seed=221, scale=True, target_encode=True, one_hot_encode=False, batch_size=128, val_batch_size=256, early_stopping_rounds=20, epochs=3, logging_period=100, num_features=14, num_classes=1, cat_idx=[1, 3, 5, 6, 7, 8, 9, 13], cat_dims=[9, 16, 7, 15, 6, 5, 2, 42])
Train model with given hyperparameters
Loading dataset Adult...
Dataset loaded!
(32561, 14)
Scaling the data...
Traceback (most recent call last):
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/train.py", line 154, in <module>
    main_once(arguments)
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/train.py", line 138, in main_once
    sc, time = cross_validation(model, X, y, args)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/train.py", line 44, in cross_validation
    loss_history, val_loss_history = curr_model.fit(X_train, y_train, X_test, y_test)  # X_val, y_val)
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/models/stochastic_gates.py", line 34, in fit
    loss, val_loss = self.model.fit(X, y, nr_epochs=self.args.epochs, valid_X=X_val, valid_y=y_val,
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/models/stg_lib/stg.py", line 177, in fit
    return self.train(data_loader, nr_epochs, val_data_loader, verbose, meters, early_stop, print_interval)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/models/stg_lib/stg.py", line 234, in train
    self.train_epoch(data_loader, meters=meters)
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/models/stg_lib/stg.py", line 215, in train_epoch
    self.train_step(feed_dict, meters=meters)
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/models/stg_lib/stg.py", line 138, in train_step
    loss = as_float(loss)
           ^^^^^^^^^^^^^^
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/models/stg_lib/utils.py", line 266, in as_float
    return stmap(_as_float, obj)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/models/stg_lib/utils.py", line 215, in stmap
    elif isinstance(iterable, (collections.Sequence, collections.UserList)):
                               ^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'collections' has no attribute 'Sequence'


----------------------------------------------------------------------------
Training SAINT with config/adult.yml in env torch

WARNING:root:imports error 
 You need to install pymongo>=3.9.0 in order to use MongoOutput 
/home/marwan.housni/.conda/envs/torch/lib/python3.12/site-packages/torch/utils/data/dataloader.py:558: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 1, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
Namespace(config='config/adult.yml', model_name='SAINT', dataset='Adult', objective='binary', use_gpu=False, gpu_ids=[0, 1], data_parallel=True, optimize_hyperparameters=False, n_trials=2, direction='maximize', num_splits=5, shuffle=True, seed=221, scale=True, target_encode=True, one_hot_encode=False, batch_size=128, val_batch_size=256, early_stopping_rounds=20, epochs=3, logging_period=100, num_features=14, num_classes=1, cat_idx=[1, 3, 5, 6, 7, 8, 9, 13], cat_dims=[9, 16, 7, 15, 6, 5, 2, 42])
Train model with given hyperparameters
Loading dataset Adult...
Dataset loaded!
(32561, 14)
Scaling the data...
Using dim 32 and batch size 128
Using dim 32 and batch size 128
Epoch 0 loss 0.34237486124038696
Epoch 1 loss 0.32136616110801697
Epoch 2 loss 0.3169342577457428
/home/marwan.housni/.conda/envs/torch/lib/python3.12/site-packages/torch/utils/data/dataloader.py:558: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 1, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
{'Log Loss - mean': 0.3156974859129537, 'Log Loss - std': 0.0, 'AUC - mean': 0.9082308857289597, 'AUC - std': 0.0, 'Accuracy - mean': 0.8453861507753724, 'Accuracy - std': 0.0, 'F1 score - mean': 0.8453861507753723, 'F1 score - std': 0.0}
Using dim 32 and batch size 128
Epoch 0 loss 0.32982009649276733
Epoch 1 loss 0.30986034870147705
Epoch 2 loss 0.30631572008132935
/home/marwan.housni/.conda/envs/torch/lib/python3.12/site-packages/torch/utils/data/dataloader.py:558: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 1, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
{'Log Loss - mean': 0.3109592008247092, 'Log Loss - std': 0.004738285088244559, 'AUC - mean': 0.9114438991256251, 'AUC - std': 0.0032130133966654095, 'Accuracy - mean': 0.851440004134615, 'Accuracy - std': 0.006053853359242578, 'F1 score - mean': 0.8514400041346148, 'F1 score - std': 0.006053853359242634}
Using dim 32 and batch size 128
Epoch 0 loss 0.3197258710861206
Epoch 1 loss 0.3018304705619812
Epoch 2 loss 0.29893168807029724
/home/marwan.housni/.conda/envs/torch/lib/python3.12/site-packages/torch/utils/data/dataloader.py:558: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 1, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
{'Log Loss - mean': 0.30705055361709177, 'Log Loss - std': 0.006747044516345739, 'AUC - mean': 0.9138689722210595, 'AUC - std': 0.0043179002336589944, 'Accuracy - mean': 0.8540722058686131, 'Accuracy - std': 0.006187869768108198, 'F1 score - mean': 0.8540722058686131, 'F1 score - std': 0.00618786976810825}
Using dim 32 and batch size 128
Epoch 0 loss 0.3288508951663971
Epoch 1 loss 0.3121880888938904
Epoch 2 loss 0.3074205219745636
/home/marwan.housni/.conda/envs/torch/lib/python3.12/site-packages/torch/utils/data/dataloader.py:558: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 1, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
{'Log Loss - mean': 0.3070756605982203, 'Log Loss - std': 0.005843273770812686, 'AUC - mean': 0.9138163827244268, 'AUC - std': 0.0037405205234489138, 'Accuracy - mean': 0.8557722133695187, 'Accuracy - std': 0.006114521707354329, 'F1 score - mean': 0.8557722133695187, 'F1 score - std': 0.006114521707354375}
Using dim 32 and batch size 128
Epoch 0 loss 0.32951512932777405
Epoch 1 loss 0.31720685958862305
Epoch 2 loss 0.31007733941078186
{'Log Loss - mean': 0.30800705017820074, 'Log Loss - std': 0.005548425442169955, 'AUC - mean': 0.9133264296472781, 'AUC - std': 0.0034861742785328637, 'Accuracy - mean': 0.8555330041108483, 'Accuracy - std': 0.005489880221416518, 'F1 score - mean': 0.8555330041108483, 'F1 score - std': 0.005489880221416554}
Results: {'Log Loss - mean': 0.30800705017820074, 'Log Loss - std': 0.005548425442169955, 'AUC - mean': 0.9133264296472781, 'AUC - std': 0.0034861742785328637, 'Accuracy - mean': 0.8555330041108483, 'Accuracy - std': 0.005489880221416518, 'F1 score - mean': 0.8555330041108483, 'F1 score - std': 0.005489880221416554}
Train time: 43.478036599
Inference time: 0.8048455895999922
Finished cross validation
{'Log Loss - mean': 0.30800705017820074, 'Log Loss - std': 0.005548425442169955, 'AUC - mean': 0.9133264296472781, 'AUC - std': 0.0034861742785328637, 'Accuracy - mean': 0.8555330041108483, 'Accuracy - std': 0.005489880221416518, 'F1 score - mean': 0.8555330041108483, 'F1 score - std': 0.005489880221416554}
(43.478036599, 0.8048455895999922)


----------------------------------------------------------------------------
Training RandomForest with config/adult.yml in env sklearn

WARNING:root:imports error 
 You need to install pymongo>=3.9.0 in order to use MongoOutput 
Namespace(config='config/adult.yml', model_name='RandomForest', dataset='Adult', objective='binary', use_gpu=False, gpu_ids=[0, 1], data_parallel=True, optimize_hyperparameters=False, n_trials=2, direction='maximize', num_splits=5, shuffle=True, seed=221, scale=True, target_encode=True, one_hot_encode=False, batch_size=128, val_batch_size=256, early_stopping_rounds=20, epochs=3, logging_period=100, num_features=14, num_classes=1, cat_idx=[1, 3, 5, 6, 7, 8, 9, 13], cat_dims=[9, 16, 7, 15, 6, 5, 2, 42])
Train model with given hyperparameters
Loading dataset Adult...
Dataset loaded!
(32561, 14)
Scaling the data...
{'Log Loss - mean': 0.30977761330022024, 'Log Loss - std': 0.0, 'AUC - mean': 0.9111400135307671, 'AUC - std': 0.0, 'Accuracy - mean': 0.8519883310302472, 'Accuracy - std': 0.0, 'F1 score - mean': 0.8519883310302472, 'F1 score - std': 0.0}
{'Log Loss - mean': 0.30416757162331975, 'Log Loss - std': 0.00561004167690049, 'AUC - mean': 0.9153498341999983, 'AUC - std': 0.004209820669231146, 'Accuracy - mean': 0.8583498166207747, 'Accuracy - std': 0.006361485590527516, 'F1 score - mean': 0.8583498166207748, 'F1 score - std': 0.006361485590527571}
{'Log Loss - mean': 0.29937940087274956, 'Log Loss - std': 0.008175259629981477, 'AUC - mean': 0.9186849269545571, 'AUC - std': 0.005836158617425154, 'Accuracy - mean': 0.8609309997783052, 'Accuracy - std': 0.006348544105889996, 'F1 score - mean': 0.8609309997783052, 'F1 score - std': 0.006348544105890048}
{'Log Loss - mean': 0.2999609937176581, 'Log Loss - std': 0.007151286823009551, 'AUC - mean': 0.9180800837934969, 'AUC - std': 0.005161692193869498, 'Accuracy - mean': 0.8614537780892572, 'Accuracy - std': 0.0055720643109648665, 'F1 score - mean': 0.8614537780892572, 'F1 score - std': 0.005572064310964906}
{'Log Loss - mean': 0.3004481364473847, 'Log Loss - std': 0.0064700815092595294, 'AUC - mean': 0.9175920519969862, 'AUC - std': 0.004718808449282955, 'Accuracy - mean': 0.8617981576065409, 'Accuracy - std': 0.005031173775186157, 'F1 score - mean': 0.8617981576065409, 'F1 score - std': 0.0050311737751861894}
Results: {'Log Loss - mean': 0.3004481364473847, 'Log Loss - std': 0.0064700815092595294, 'AUC - mean': 0.9175920519969862, 'AUC - std': 0.004718808449282955, 'Accuracy - mean': 0.8617981576065409, 'Accuracy - std': 0.005031173775186157, 'F1 score - mean': 0.8617981576065409, 'F1 score - std': 0.0050311737751861894}
Train time: 1.5839270274000001
Inference time: 0.08068738779999993
Finished cross validation
{'Log Loss - mean': 0.3004481364473847, 'Log Loss - std': 0.0064700815092595294, 'AUC - mean': 0.9175920519969862, 'AUC - std': 0.004718808449282955, 'Accuracy - mean': 0.8617981576065409, 'Accuracy - std': 0.005031173775186157, 'F1 score - mean': 0.8617981576065409, 'F1 score - std': 0.0050311737751861894}
(1.5839270274000001, 0.08068738779999993)


----------------------------------------------------------------------------
Training LinearModel with config/adult.yml in env sklearn

WARNING:root:imports error 
 You need to install pymongo>=3.9.0 in order to use MongoOutput 
/home/marwan.housni/.conda/envs/sklearn/lib/python3.12/site-packages/sklearn/linear_model/_logistic.py:460: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(
/home/marwan.housni/.conda/envs/sklearn/lib/python3.12/site-packages/sklearn/linear_model/_logistic.py:460: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(
/home/marwan.housni/.conda/envs/sklearn/lib/python3.12/site-packages/sklearn/linear_model/_logistic.py:460: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(
/home/marwan.housni/.conda/envs/sklearn/lib/python3.12/site-packages/sklearn/linear_model/_logistic.py:460: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(
/home/marwan.housni/.conda/envs/sklearn/lib/python3.12/site-packages/sklearn/linear_model/_logistic.py:460: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(
Namespace(config='config/adult.yml', model_name='LinearModel', dataset='Adult', objective='binary', use_gpu=False, gpu_ids=[0, 1], data_parallel=True, optimize_hyperparameters=False, n_trials=2, direction='maximize', num_splits=5, shuffle=True, seed=221, scale=True, target_encode=True, one_hot_encode=False, batch_size=128, val_batch_size=256, early_stopping_rounds=20, epochs=3, logging_period=100, num_features=14, num_classes=1, cat_idx=[1, 3, 5, 6, 7, 8, 9, 13], cat_dims=[9, 16, 7, 15, 6, 5, 2, 42])
Train model with given hyperparameters
Loading dataset Adult...
Dataset loaded!
(32561, 14)
Scaling the data...
{'Log Loss - mean': 0.39490576157734353, 'Log Loss - std': 0.0, 'AUC - mean': 0.8458238968609033, 'AUC - std': 0.0, 'Accuracy - mean': 0.8186703516044833, 'Accuracy - std': 0.0, 'F1 score - mean': 0.8186703516044833, 'F1 score - std': 0.0}
{'Log Loss - mean': 0.38835866418498166, 'Log Loss - std': 0.006547097392361895, 'AUC - mean': 0.8503069355515136, 'AUC - std': 0.0044830386906103414, 'Accuracy - mean': 0.8217276819447479, 'Accuracy - std': 0.003057330340264508, 'F1 score - mean': 0.8217276819447479, 'F1 score - std': 0.003057330340264508}
{'Log Loss - mean': 0.3849703975611493, 'Log Loss - std': 0.007178929186984203, 'AUC - mean': 0.8532723065791764, 'AUC - std': 0.0055664418255960075, 'Accuracy - mean': 0.8237705430819204, 'Accuracy - std': 0.0038181246432257014, 'F1 score - mean': 0.8237705430819204, 'F1 score - std': 0.0038181246432257014}
{'Log Loss - mean': 0.3842774307990331, 'Log Loss - std': 0.006331933117460542, 'AUC - mean': 0.8536908092141524, 'AUC - std': 0.0048748732614042395, 'Accuracy - mean': 0.8240241603826934, 'Accuracy - std': 0.0033356441730435955, 'F1 score - mean': 0.8240241603826934, 'F1 score - std': 0.0033356441730435955}
{'Log Loss - mean': 0.3844845996024516, 'Log Loss - std': 0.005678589371986169, 'AUC - mean': 0.8532926492701398, 'AUC - std': 0.004432339880454571, 'Accuracy - mean': 0.8244220310088574, 'Accuracy - std': 0.0030877859025496552, 'F1 score - mean': 0.8244220310088574, 'F1 score - std': 0.003087785902549644}
Results: {'Log Loss - mean': 0.3844845996024516, 'Log Loss - std': 0.005678589371986169, 'AUC - mean': 0.8532926492701398, 'AUC - std': 0.004432339880454571, 'Accuracy - mean': 0.8244220310088574, 'Accuracy - std': 0.0030877859025496552, 'F1 score - mean': 0.8244220310088574, 'F1 score - std': 0.003087785902549644}
Train time: 0.19721391680000006
Inference time: 0.004252609000000085
Finished cross validation
{'Log Loss - mean': 0.3844845996024516, 'Log Loss - std': 0.005678589371986169, 'AUC - mean': 0.8532926492701398, 'AUC - std': 0.004432339880454571, 'Accuracy - mean': 0.8244220310088574, 'Accuracy - std': 0.0030877859025496552, 'F1 score - mean': 0.8244220310088574, 'F1 score - std': 0.003087785902549644}
(0.19721391680000006, 0.004252609000000085)


----------------------------------------------------------------------------
Training ModelTree with config/adult.yml in env gbdt

WARNING:root:imports error 
 You need to install pymongo>=3.9.0 in order to use MongoOutput 
Namespace(config='config/adult.yml', model_name='ModelTree', dataset='Adult', objective='binary', use_gpu=False, gpu_ids=[0, 1], data_parallel=True, optimize_hyperparameters=False, n_trials=2, direction='maximize', num_splits=5, shuffle=True, seed=221, scale=True, target_encode=True, one_hot_encode=False, batch_size=128, val_batch_size=256, early_stopping_rounds=20, epochs=3, logging_period=100, num_features=14, num_classes=1, cat_idx=[1, 3, 5, 6, 7, 8, 9, 13], cat_dims=[9, 16, 7, 15, 6, 5, 2, 42])
Train model with given hyperparameters
Loading dataset Adult...
Dataset loaded!
(32561, 14)
Scaling the data...
Traceback (most recent call last):
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/train.py", line 154, in <module>
    main_once(arguments)
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/train.py", line 138, in main_once
    sc, time = cross_validation(model, X, y, args)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/train.py", line 44, in cross_validation
    loss_history, val_loss_history = curr_model.fit(X_train, y_train, X_test, y_test)  # X_val, y_val)
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/models/modeltree.py", line 28, in fit
    X = np.array(X, dtype=np.float)
                          ^^^^^^^^
  File "/home/marwan.housni/.conda/envs/gbdt/lib/python3.11/site-packages/numpy/__init__.py", line 324, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations. Did you mean: 'cfloat'?


----------------------------------------------------------------------------
Training NAM with config/adult.yml in env torch

WARNING:root:imports error 
 You need to install pymongo>=3.9.0 in order to use MongoOutput 
Namespace(config='config/adult.yml', model_name='NAM', dataset='Adult', objective='binary', use_gpu=False, gpu_ids=[0, 1], data_parallel=True, optimize_hyperparameters=False, n_trials=2, direction='maximize', num_splits=5, shuffle=True, seed=221, scale=True, target_encode=True, one_hot_encode=False, batch_size=128, val_batch_size=256, early_stopping_rounds=20, epochs=3, logging_period=100, num_features=14, num_classes=1, cat_idx=[1, 3, 5, 6, 7, 8, 9, 13], cat_dims=[9, 16, 7, 15, 6, 5, 2, 42])
Train model with given hyperparameters
Loading dataset Adult...
Dataset loaded!
(32561, 14)
Scaling the data...
Traceback (most recent call last):
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/train.py", line 154, in <module>
    main_once(arguments)
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/train.py", line 133, in main_once
    model_name = str2model(args.model_name)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/models/__init__.py", line 81, in str2model
    from models.neural_additive_models import NAM
  File "/srv/lustre01/project/manapy-um6p-st-msda-1wabcjwe938/users/marwan.housni/TabSurvey/models/neural_additive_models.py", line 4, in <module>
    from nam.config import defaults
ModuleNotFoundError: No module named 'nam'


----------------------------------------------------------------------------
Training LightGBM with config/adult.yml in env gbdt

slurmstepd-slurm-compute-h21a8-u7-svn4: error: *** JOB 2319765 ON slurm-compute-h21a8-u7-svn4 CANCELLED AT 2024-03-18T15:34:38 ***
